tests/model/test_time_to_triage.py

```diff
@@ -0,0 +1,96 @@
+import copy
+import datetime
+
+from treeherder.perf.models import PerformanceAlertSummary
+
+
+class TestTimeToClass:
```

Comment:
You need to test also the creation of objects, to check if the triage date is updated to newly created `PerformanceAlertSummary `objects using `PerformanceAlertSummary.objects.create()`

Reply:
I still think this is important to add to your PR.